src/App.js

```diff
@@ -21,6 +22,8 @@ function App() {
         <Route path="/history" element={<History />} />
         <Route path="/watchlater" element={<WatchLater />} />
         <Route path="/liked" element={<Liked />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/signup" element={<Signup />} />
         <Route path="/mock" element={<MockAPI />} />
         {/* <Route path="*" element */}
       </Routes>
```

Comment:
Can make separate file for routing and then import here

Reply:
yes thanks